src/api/python/cvc5.pxi

```diff
@@ -1842,6 +1846,10 @@ cdef class Solver:
 
             ( declare-sort <symbol> <numeral> )
 
+        .. note::
+          This corresponds to mkUninterpretedSort() if arity = 0, and to
+          mkUninterpretedSortConstructorSort() if arity > 0.
```

Comment:
```suggestion
          mkUninterpretedSortConstructorSort(arity) if arity > 0.
```

Reply:
Cannot be cross-referenced when arguments are given, won't fix.